README.md

```diff
@@ -1,6 +1,9 @@
 # TheRock
 
-The HIP Environment and ROCm Kit - A lightweight open source build system for HIP and ROCm
+The HIP Environment and ROCm Kit - A lightweight open source build system for HIP and ROCm.
+
+We are currently in an **early preview state** but welcome contributors. Come try us out!
+Please see [CONTRIBUTING.md](CONTRIBUTING.md) for more info.
```

Comment:
I think we should also include more information here about what the project is and what it will enable.

Brainstorming:

* Link to the "official" release channels and establish that this project enables building a similar set of artifacts from public sources for a configurable set of targets, operating systems, etc.
* Add tested support tables, like https://rocm.docs.amd.com/en/latest/compatibility/compatibility-matrix.html
* Link to published artifacts

Reply:
See also https://github.com/ROCm/TheRock/issues/136

Reply:
Let me pull the artifact doc out of cold storage and land it.